### PR TITLE
array: add how to create an empty array

### PR DESCRIPTION
**Pull request by dhonx**
People who want to learn V may not all have learned Go and may not yet have experience in the programming world. So I think this will help them on how to make a new, empty array.

### CODE REVIEW

File: examples/arrays/arrays.md [hunk 1]
```diff
@@ -72,6 +72,15 @@ println('Thiago' in names)
 >> true
 ```
 
+## Create an empty array
```
Comment (vbrazo) [inline examples/arrays/arrays.md]: I think this piece could probably be in the `How to declare an array` section.

Comment (dhonx) [reply examples/arrays/arrays.md]: ya.. great idea !!! :+1:

Comment (dhonx) [reply examples/arrays/arrays.md]: im using header level 3

Comment (Delta456) [reply examples/arrays/arrays.md]: Agreed with @vbrazo

File: examples/arrays/arrays.md [hunk 2]
```diff
@@ -22,6 +22,15 @@ println(github_users[0])
 >> vbrazo
 ```
 
+### Create an empty array
```
Comment (Delta456) [inline examples/arrays/arrays.md:25]: Looks fine

**Review by Delta456 — APPROVED** [review]

**Review by vbrazo — APPROVED** [review]
Thanks man 💥 

You're great. Let's keep this up!

I'll push a new example over the next days.

**Pull request merged**